swagger.yaml

```diff
@@ -368,15 +368,77 @@ definitions:
         $ref: "#/definitions/Version"
       package:
         $ref: "#/definitions/Package"
+  SnykCVEPremium:
+    title: CVE details from Snyk avilable for signed-in Users
+    description: Snyk CVE details
+    properties:
+      malicious:
+        type: boolean
+      patch_exists:
+        type: boolean
+      fixable:
+        type: boolean
+      url:
+        type: string
+      exploit:
+        type: string
+      description:
+        type: string
+      fixed_in:
+        type: string
+      references:
+        type: array
+        items:
+          type: object
+          properties:
+            title:
+              type: string
+            url:
+              type: string
+  SnykCVE:
+    title: CVE details from Snyk
+    description: Snyk CVE details
+    properties:
+      cve_ids:
+        type: array
+        items:
+          type: string
+      cvss:
+        type: number
+        format: float
+      cvss_v3:
+        type: string
+      cwes:
+        type: array
+        items:
+          type: string
+      id:
+        type: string
+      severity:
+        type: string
+        enum:
+          - low
+          - medium
+          - high
+      title:
+        type: string
+      url:
+        type: string
+      private:
+        type: boolean
+      premium_details:
+        $ref: "#/definitions/SnykCVEPremium"
   CVE:
```

Comment:
I see definition updates only. 
How are you planning to differentiate b/w API calls from VS Code v0.13 and v.14?

Reply:
It is for stack report UI which is a hosted Angular App., it is not tied with vscode plugin. Following incremental updates will be done to ensure smooth transition,

- Modify stack report API to return both CVE model and SnykCVE model using Snyk data, this ensures existing stack report UI won't break abruptly.
- Stack report UI will be modified to accommodate snyk specific changes

Reply:
On similar cases, In Component Analyses, We are passing `security-vendor` header in API calls. 
Can we maintain the same practice here..?

Reply:
Thanks, I added the same. Please check.